airbyte-integrations/connectors/source-mysql/src/test/java/io/airbyte/integrations/source/mysql/MySqlSourceTests.java

```diff
@@ -86,4 +94,25 @@ private JsonNode buildConfigEscapingNeeded() {
         JdbcUtils.DATABASE_KEY, "db/foo"));
   }
 
+  @Test
+  void testCheckWithSSlModeDisabled() throws Exception {
```

Comment:
> This isn't quite what I was thinking, my statement was more that these should be within the `strict-encrypt` tests because this enforcement of `SSL` or `SSH Tunnel` is not a requirement for OSS users so the `check` operation should always be successful
> 
> Currently the tests for failed `check` when SSH tunnel isn't established already exist for `source-mysql-strict-encrypt` and your test should probably live there and create a corresponding test for `source-postgres-strict-encrypt`
> 
> cc: @rodireich since you wrote these tests in case Vitalii needs more guidance
> 
> https://github.com/airbytehq/airbyte/blob/1cdf1ba9e8b75846a72fd9d8b3271f18eca240ad/airbyte-integrations/connectors/source-mysql-strict-encrypt/src/test/java/io/airbyte/integrations/source/mysql_strict_encrypt/MySqlStrictEncryptJdbcSourceAcceptanceTest.java#L240-L257

@ryankfu
1. Test for _source-mysql-strict-encrypt_ i added to MySqlStrictEncryptJdbcSourceAcceptanceTest
2. For _source-postgres-strict-encrypt_, I can't do the same because the PostgresStrictEncryptJdbcSourceAcceptanceTest was previously removed from the _source-postgres-strict-encrypt_ module as part of a future migration and merging the two modules into one.
At the moment, I have not found a more suitable place than to create a new PostgresSourceStrictEncryptTest class where I added my tests, please take a look, the similar test case that the user encountered is also described there

Regarding the tests that @rodireich did - they do not apply to this task, since this task concerns exclusively to enforcing ssl, provided that the server does not support ssl, but I agree that similar tests like we have [here]([airbyte/airbyte-integrations/connectors/source-mysql-strict-encrypt/src/test/java/io/airbyte/integrations/source/mysql_strict_encrypt/MySqlStrictEncryptJdbcSourceAcceptanceTest.java](https://github.com/airbytehq/airbyte/blob/1cdf1ba9e8b75846a72fd9d8b3271f18eca240ad/airbyte-integrations/connectors/source-mysql-strict-encrypt/src/test/java/io/airbyte/integrations/source/mysql_strict_encrypt/MySqlStrictEncryptJdbcSourceAcceptanceTest.java#L240-L257)) should be added for _source-postgres-strict-encrypt_ connector as well, but within the scope of another task